### DISCUSSION TITLE
Dragon longsword 2009 version

### DISCUSSION THREAD

**Discussion by ToTheHorizon** [runelite/runelite#15486] [Ideas]
Hey, I'm wondering if it's possible to make it so you can change the design for the dragon longsword to the 2009 version? This:

https://runescape.wiki/images/Dragon_longsword_detail_old.png?312fa

I play the game in 2008 HD mode and the the original dragon longsword sticks out like a sword thumb in my opinion whereas I really love the 2009 updated version, and it would be nice if it could be optional to make your dragon longsword look like that. Is there code available for the 2009 version?

Thanks,

ToTheHorizon

**Comment by Nightfirecat** [comment]
We don't have a way to replace models and animations in general, let alone one specific model and its associated animations.
